adoc/quickstart3_chap_suma_setup_with_yast.adoc

```diff
@@ -17,6 +17,14 @@ You will perform the following procedures:
 [[suma-setup-with-yast-sumasetup]]
 == {productname} Setup
 
+[WARNING]
+.Third Party Software Not Allowed
+--
+{productname} is a extension of {sls} and thus compatible the with software shipped with {sls}.
```

Comment:
> {productname} is an extension of {sls}, so it is compatible with all {sls} default software.

(Or similar. There's a couple of typos in there, plus I really dislike 'thus' :P )